docs/tutorials/configure/build-a-mock-robot.md

```diff
@@ -445,21 +445,22 @@ func toggleMotor (ctx context.Context, m motor.Motor) {
 {{< /tabs >}}
 
 Now, invoke your new function in `main()`.
+You can also comment out invoking `randomMovement()`.
```

Comment:
Opt nit: minor clarification to highlight why disabling might be useful:

```suggestion
Optionally, you can comment out the `randomMovement()` invocation to focus on testing the subpart.
```

You might want to even directly instruct users to comment it out then, since the following GIF demonstrates usage without it (guess: because no `values:` are printed)